src/components/views/AdvancedView.js

```
@@ -271,86 +271,403 @@ export class AdvancedView extends LitElement {
+    }
+
+    // Get max allowed tokens for current model
+    getMaxAllowedTokens() {
+        return AdvancedView.MODEL_MAX_TOKENS[this.selectedModel] || 65536;
+    }
+
+    // Get default temperature (same for all models)
+    getDefaultTemperature() {
+        return AdvancedView.DEFAULT_TEMPERATURE;
+    }
+
```

Biome (2.1.2)

[error] Duplicate class member name "getDefaultTemperature"



(lint/suspicious/noDuplicateClassMembers)